h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -99,6 +100,14 @@ public void initMetalearnerParams(Metalearner.Algorithm algo) {
     
     public final Frame blending() { return _blending == null ? null : _blending.get(); }
 
+    @Override
+    public String[] getNonPredictors() {
```

Comment:
I know this method is not new but this is the first time I noticed it:

Model builder defines methods numSpecialCols and specialColNames, I think the naming needs to be unified.

Reply:
@michalkurka I think I created it for TE. I probably didn't notice the `specialCol__` properties because those are available only at ModelBuilder level, whereas I needed those also for scoring. Another small difference is that it includes the response column, but I still agree that it should be unified.
Creating a ticket for this.

Reply:
https://h2oai.atlassian.net/browse/PUBDEV-8078